web/src/user_guide/Page.svelte

```diff
@@ -0,0 +1,518 @@
+  import "@picocss/pico/css/pico.conditional.jade.min.css";
+  import Toc from "svelte-toc";
+  import About from "../About.svelte";
+  import image01 from "./image01.webp?url";
+  import image02 from "./image02.webp?url";
+  import image03 from "./image03.webp?url";
+  import image04 from "./image04.webp?url";
+  import image05 from "./image05.webp?url";
+  import image06 from "./image06.webp?url";
+  import image07 from "./image07.webp?url";
+  import image08 from "./image08.webp?url";
+  import image09 from "./image09.webp?url";
+  import image10 from "./image10.webp?url";
+  import image11 from "./image11.webp?url";
+  import image12 from "./image12.webp?url";
+  import image13 from "./image13.webp?url";
+  import image14 from "./image14.webp?url";
+  import image15 from "./image15.webp?url";
+  import image16 from "./image16.webp?url";
+  import image17 from "./image17.webp?url";
+  import image18 from "./image18.webp?url";
+  import image19 from "./image19.webp?url";
+  import image20 from "./image20.webp?url";
+  import image21 from "./image21.webp?url";
+  import image22 from "./image22.webp?url";
+  import image23 from "./image23.webp?url";
+  import image24 from "./image24.webp?url";
+  import image25 from "./image25.webp?url";
+  import image26 from "./image26.webp?url";
+  import image27 from "./image27.webp?url";
+  import image28 from "./image28.webp?url";
+  import image29 from "./image29.webp?url";
+  import image30 from "./image30.webp?url";
+  import image31 from "./image31.webp?url";
+</script>
+
+<div class="container">
+  <div class="contents">
+    <h1>User guide for the Connected Neighbourhood Tool</h1>
+
+    <h2 id="overview">Overview</h2>
+    <p>
+      The Connected Neighbourhood Tool helps you design a Low-Traffic
+      Neighbourhood (LTN). This guide is written specifically for the Scottish
+      version of the tool, but most of it applies elsewhere too. In this guide,
+      we assume you are already familiar with the concept and purpose of LTNs.
+      If you have any trouble using the tool, please email the maintainer at <a
+        href="mailto:dabreegster@gmail.com">dabreegster@gmail.com</a
+      >.
+    </p>
+    <p>The overall process of using the CNT looks like this:</p>
+    <ol>
+      <li>Choose your study area</li>
+      <li>Add one or more neighbourhood boundaries</li>
+      <li>
+        Design your LTN with modal filters, one-way streets, turn restrictions,
+        and sometimes by reclassifying main roads
+      </li>
+      <li>Explore the effects of your proposal</li>
+      <li>Share your work with a colleague</li>
+    </ol>
+
+    <h2 id="adding-a-neighbourhood">Adding a neighbourhood</h2>
+    <p>
+      Start using the tool by going to <a href="https://cnt.scot/"
+        >https://cnt.scot/</a
+      >
+      and picking your study area on the map or from the list. The study areas are
+      defined by Local Authority Districts.
+    </p>
+    <img alt="" src={image01} />
```

Comment:
No alt text anywhere :\  There's 31 images to possibly caption, and the prose surrounding the image makes them clear in many cases. Could write captions in the future.

Reply:
Maybe better to remove the empty alt tag? I'm not sure if that might confuse screen readers.